qemu/tests/block_commit_stress.py

```diff
@@ -0,0 +1,98 @@
+import logging
+
+from virttest import utils_misc
+from virttest import utils_test
+from virttest import error_context
+
+from qemu.tests import blk_commit
+
+
+class BlockCommitStress(blk_commit.BlockCommit):
+
+    @error_context.context_aware
+    def load_stress(self):
+        """
+        load IO/CPU/Memoery stress in guest
+        """
+        error_context.context("launch stress app in guest", logging.info)
+        args = (self.test, self.params, self.env, self.params["stress_test"])
+        bg_test = utils_test.BackgroundTest(utils_test.run_virt_sub_test, args)
+        bg_test.start()
+        if not utils_misc.wait_for(bg_test.is_alive, first=10, step=3, timeout=100):
+            self.test.fail("background test start failed")
+        if not utils_misc.wait_for(self.app_running, timeout=360, step=5):
+            self.test.fail("stress app isn't running")
+
+    @error_context.context_aware
+    def unload_stress(self):
+        """
+        stop stress app
+        """
+        def _unload_stress():
+            session = self.get_session()
+            cmd = self.params.get("stop_cmd")
+            session.sendline(cmd)
+            return self.app_running()
+
+        error_context.context("stop stress app in guest", logging.info)
+        stopped = utils_misc.wait_for(_unload_stress, first=2.0,
+                                      text="wait stress app quit", step=1.0,
+                                      timeout=self.params["wait_timeout"])
+        if not stopped:
+            logging.warn("stress app is still running")
```

Comment:
no need to use test.error since stress app still running will not affect test result i think. it should be a warning at most